ffi/capi/src/pluralrules.rs

```diff
@@ -130,6 +130,14 @@ pub mod ffi {
             ))))
         }
 
+        /// Construct for a given integer
+        #[diplomat::attr(all(supports = fallible_constructors, supports = named_constructors), named_constructor)]
+        pub fn from_integer(int: i64) -> Box<PluralOperands> {
```

Comment:
this should be `from_int64`, using the C/C++ type name, with renames for Dart and JS. See the `Decimal` constructors